dataframe-excel/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/XlsxTest.kt

```diff
@@ -109,4 +109,19 @@ class XlsxTest {
             DataFrame.readExcel(testResource("xlsx6.xlsx"), skipRows = 4)
         }
     }
+
+    @Test
+    fun `write to new sheet when keepFile is true`() {
+        val names = (1..5).map { "column$it" }
+        val df = dataFrameOf(names).randomDouble(7)
+        val fileLoc = testResource("generated_wb.xlsx").toURI().toString().removeRange(0, 6)
```

Comment:
This test is failing with NPE =( Let's use
Files.createTempFile("excel", ".xlsx").toFile()

Reply:
Updated, thanks for the tip!